### PR TITLE
Signup Flow Fixes

### PR DESCRIPTION
**Pull request by nwestallen**
## Description

- replaces axiosWithAuth with axios call to fix signup bug
- resets state of pending profiles on manage profiles correctly to fix rerender
- adds antD success & error notifications to signup page

Fixes # (issue)

- request access errors out on signup page
- all pending profiles disappear on reject

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: src/components/pages/Login/SignupPage.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
 import React, { useState } from 'react';
 import { Form, Input, Button } from 'antd';
-import axiosWithAuth from '../../../utils/axiosWithAuth';
 import './_SignupPageStyled.less';
 import logo from '../../../styles/hrf-logo.png';
+import axios from 'axios';
+import { notification } from 'antd';
```
Comment (katieolson84) [inline src/components/pages/Login/SignupPage.js:6]: Love the notification addition as well as fixing axios here

File: src/components/pages/Login/SignupPage.js [hunk 2]
```diff
@@ -13,15 +15,41 @@ const initialFormValues = {
...
+      top: 128,
+      icon: <CheckCircleOutlined style={{ color: 'green' }} />,
+    });
+  };
+
+  const failNotification = () => {
+    notification.open({
+      message: 'Error',
+      description:
+        'There was an issue with the signup. Please try again and if the issue persists contact the site administrator.',
```
Comment (katieolson84) [inline src/components/pages/Login/SignupPage.js:31]: Love this message, nice work!

**Review by katieolson84 — APPROVED** [review]
Everything looks great, no changes needed. Nice fix!

**Review by dgamboa — APPROVED** [review]
Great notifications -- will make the UX much better

**Pull request merged**